www/utils/packages/typedoc-generate-references/src/constants/references-details.ts

```diff
@@ -22,6 +22,16 @@ export const customModulesOptions: Record<string, Partial<TypeDocOptions>> = {
       "packages/core/types/src/stock-location/service.ts"
     ),
   },
+  "inventory-next-models": {
```

Comment:
Q: should these have "next"?

Reply:
I left them as `next` because otherwise there will be huge renames in the generated files in the docs, which would result in infinity changes again 😅 we can look into renaming them at a later point when we have fewer changes or with some manual work.